specification.md

```diff
@@ -454,6 +454,54 @@ to `null`.
 
 
 
+<a name="jobexecutor-list"></a>
+```java
+List<String> list()
+```
+
+Return a list of native job IDs which are known to this executor instance.  
+The returned list MAY contain IDs of jobs which were not submitted via this
+instance, and MAY be missing IDs of jobs which have been submitted by this
+instance but are finalized and purged already.  The returned job IDs MUST be
+uniquely identify job in the scope of the `JobExecutor` instance and thus in the
+scope of the backend that `JobExecutor` is bound to, but should not be assumed
+to be unique beyond that scope.
+
+IDs for any job which has been submitted via this instance and which is not yet
+in a final state MUST be returned.  Information for jobs in final state may get
+purged by the backend, and the implementation MAY purge that information also.
+The native IDs for those jobs thus MAY NOT be returned by this call, even if the
+application still holds handles to those jobs (and could thus retrieve the
+native job ID directly).
+
+
+The returned IDs can be used to re-attach a `Job` instance to the backend job
+via the `executor.attach(job, nativeJobID)` call.  This implies that the call
+SHALL only return those IDs to which the callee can attach under currently used
+authorization.
+
+
+
+<a name="jobexecutor-attach"></a>
+```java
+Job attach(Job job, String nativeJobID)
+```
+
+This method will accept a job instance in `NEW` state and a native job ID.  The
+executor will attach the Job instance to the backend job identified by that
+native (backend) job ID.  The method will return immediately, the `JobExecutor`
```

Comment:
That's a bit of a run-on. Perhaps "... will return immediately and the..."

Reply:
Done